docs/functions.rst

```diff
@@ -567,13 +582,13 @@ Yet others are handy, general-purpose utilities.
          - 1: Stroked text -- equivalent to `1 Tr`, only the character borders are shown.
          - 3: Ignored text -- equivalent to `3 Tr` (hidden text).
 
-      3. Line width in this context is important only for processing `span["type"] != 0`: it determines the thickness of the character's border line. This value may not be provided at all with the text data. In this case, a value of 5% of the fontsize (`span["size"] * 0,05`) is generated. Often, an "artificial" bold text in PDF is created by `2 Tr`. There is no equivalent span type for this case. Instead, respective text is represented by two consecutive spans -- which are identical in every aspect, except for their types, which are 0, resp 1. It is your responsibility to handle this type of situation - in :meth:`Page.get_text`, MuPDF is doing this for you.
-      4. For data compactness, the character's unicode is provided here. Use built-in function `chr()` for the character itself.
+      3. Line width in this context is important only for processing `span["type"] != 0` i.e. characters are stroked and not filled: it determines the thickness of the character's border line. This value may not be provided at all with the text data. In this case, a value of 5% of the fontsize (`span["size"] * 0,05`) is generated. Often, an "artificial" bold text in PDF is created by `2 Tr`. There is no equivalent span type for this case. Instead, respective text is represented by two consecutive spans -- which are identical in every aspect, except for their types, which are 0, resp 1. It is your responsibility to handle this type of situation - in :meth:`Page.get_text`, MuPDF is doing this for you.
```

Comment:
while we are here can we update resp 1 to : respectively 1 ?

Reply:
All docs update LGTM aside from this one tiny tweak.

Reply:
Done: "resp" -> "respectively"